### PR TITLE
Several small fixes

### PR DESCRIPTION
**Pull request by gberaudo**

### CODE REVIEW

File: closure-util.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
-  "compiler_url": "http://dev.camptocamp.com/files/closure-compiler/compiler-20150903.zip",
-  "library_url": "https://github.com/google/closure-library/archive/b129bb.zip"
+  "See downloads at": "https://github.com/google/closure-compiler/wiki/Binary-Downloads",
+  "compiler_url": "http://dl.google.com/closure-compiler/compiler-20151015.zip",
+  "library_url": "https://github.com/google/closure-library/archive/5b25e65.zip"
```
Comment (fredj) [inline closure-util.json]: We could also remove this file and rely on the [closure-util default config](https://github.com/openlayers/closure-util/blob/master/default-config.json)

Comment (gberaudo) [inline closure-util.json]: I moved this commit to another (future) PR.

**Comment by fredj** [comment]
LGTM, please merge

**Comment by gberaudo** [comment]
Thanks for the review @fredj.

**Pull request merged**